src/Microsoft.Performance.SDK/Processing/TableConfiguration.cs

```diff
@@ -78,9 +78,9 @@ public TableConfiguration(string name)
             this.Name = name;
 
             this.columnsRO = new ReadOnlyCollection<ColumnConfiguration>(EmptyArray<ColumnConfiguration>.Instance);
-            this.columnRoles = new Dictionary<ColumnRole, Guid>();
+            this.columnRoles = new Dictionary<string, Guid>();
```

Comment:
>Dictionary<string, Guid>(); [](http://example.com/codeflow?start=35&length=27)

I believe these are ordinal, case-sensitive by default. Is that what we want here?

[https://docs.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings](https://docs.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings)

Reply:
Ah good catch! Will update!

Reply:
Unless we want them to be case sensitive?

Reply:
Yeah, I'm not sure. That's why I asked ;)

Reply:
I'm tempted to say that because we know these are being modified by hand, we should make it case insensitive. But I don't feel strongly.

Reply:
Conceptually these are keys. Are keys typically case sensitive, or is there not really a "typical" case?

Reply:
I think it's scenario specific. Dictionary allows you to specify a comparer